htdocs/core/class/menubase.class.php

```diff
@@ -667,7 +667,8 @@ public function menuLoad($mymainmenu, $myleftmenu, $type_user, $menu_handler, &$
 					if ($leftmenu == 'all') {
 						$tmpcond = preg_replace('/\$leftmenu\s*==\s*["\'a-zA-Z_]+/', '1==1', $tmpcond); // Force the part of condition on leftmenu to true
 					}
-					$perms = verifCond($tmpcond);
+					// param onlysimplestring fails with 1 or 2 in auguria
+					$perms = verifCond($tmpcond, '0');
```

Comment:
What test fails ?

Reply:
`($user->hasRight("user", "user", "write") || $user->admin) && !(isModEnabled("multicompany") && !empty($user->entity) && getDolGlobalString("MULTICOMPANY_TRANSVERSE_MODE"))`
I put the same code thn in eldy but verifcond fails